pages/dashboard/dashboard-page.js

```diff
@@ -561,7 +561,7 @@ exports.DashboardPage = class DashboardPage extends BasePage {
   async moveFileToOtherTeamViaRightClick(fileName, otherTeamName) {
     const elem = this.page.locator(`button[title="${fileName}"]`).first();
     await elem.click({ button: 'right' });
-    await this.page.waitForTimeout(500);
+    await this.moveToFileMenuItem.waitFor({ state: 'visible' });
```

Comment:
I'd use `await expect(this.moveToFileMenuItem).toBeVisible();`, as it is basically the same but more legible.

Reply:
done